Formula/scdoc.rb

```diff
@@ -0,0 +1,18 @@
+class Scdoc < Formula
+  desc "Tool designed to make the process of writing man pages more friendly"
+  homepage "https://git.sr.ht/~sircmpwn/scdoc/"
+  url "https://git.sr.ht/~sircmpwn/scdoc/snapshot/scdoc-1.4.1.tar.xz"
+  sha256 "2bc96daae0ddfb858305e550eb00a544a7a6255edd0bd169dcb2fc0c035eca92"
+
+  def install
+    system "make", "LDFLAGS=", "all"
```

Comment:
What are we overriding passing a blank `LDFLAGS` here?

Reply:
By default this includes `-static` in the LDFLAGS which as best I can tell just does not work on macOS.